src/connection/connection_sr.c

```diff
@@ -149,6 +149,9 @@ CSS_THREAD_FN css_Request_handler = NULL;
 /* This will handle closed connection errors */
 CSS_THREAD_FN css_Connection_error_handler = NULL;
 
+static char css_Exec_path[PATH_MAX];
+static char **css_Argv;
```

Comment:
명확한 의미를 가지는 이름이었으면 좋겠습니다. 아래는 예시입니다. 
```suggestion
static char css_Server_exec_path[PATH_MAX];
static char **css_Server_exec_argv;
```

Reply:
수정하였습니다.